File: DSA/Aarohi-Sharma/Day33_q1.cpp
class Solution {
  public:

    string printString(string s, char ch, int count) {
        int n=s.size();
        int i=0;
        int m=0;
        for(i=0; i<n; i++){
            if(s[i]==ch){
                m++;
            }
            if(m==count){
                break;
            }
        }
        if(m==0){
            return "";
        }
        string r=s.substr(i+1, n-i+1);
        return r;
    }
};

File: DSA/Ishita-Nigam/Day33/Day33_Q1.java
class Day33_Q1 {
    public static class Solution {
        public String printString(String s, char ch, int count) {
            String str = "";
            int i = 0;
            while (i < s.length() && count != 0) {
                if (s.charAt(i) == ch) {
                    count--;
                }
                i++;
            }

            if (count == 0) {
                str = s.substring(i);
            }

            return str.equals("") ? "" : str;
        }
    }

    public static void main(String[] args) {
        Solution sol = new Solution();
        System.out.println(sol.printString("abcde", 'c', 1)); 
        System.out.println(sol.printString("geeksforgeeks", 'e', 2)); 
        System.out.println(sol.printString("hello", 'l', 2)); 
        System.out.println(sol.printString("hello", 'z', 1)); 
    }
}
